### DISCUSSION TITLE
"There was a problem loading the local develop command."

### DISCUSSION THREAD

**Discussion by turbodude** [gatsbyjs/gatsby#36154] [Help] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

1) Gatsby is installed
"gatsby -v" in the terminal gives "Gatsby CLI version: 4.18.1". 
"gatsby -v" in the project folder gives
```
Gatsby CLI version: 4.18.1
Gatsby version: unknown
  Note: this is the Gatsby version for the site at: D:\projects\gatsby\gatsby-starter-blog
```
2) At first, "npm install" showed some network errors
```
npm ERR! code ERR_SOCKET_TIMEOUT
npm ERR! network Socket timeout
npm ERR! network This is a problem related to network connectivity.
npm ERR! network In most cases you are behind a proxy or have bad network settings.
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'
```
I don't use any proxies. Got a firewall that I tried to disable and run the command again, with no difference.

3) Then on "gatsby develop" at first I got a ".ps1 cannot be loaded" error, which I seemingly resolved by setting ExecutionPolicy to Unrestricted (followed a solution from this page https://stackoverflow.com/questions/16460163/ps1-cannot-be-loaded-because-the-execution-of-scripts-is-disabled-on-this-syste )

4) Now on "gatsby develop" or "gatsby clean" or I just get the error 
```
There was a problem loading the local develop command. Gatsby may not be installed in your site's "node_modules" directory. Perhaps you need to run "npm install"? You might need 
to delete your "package-lock.json" as well.
```
However "gatsby new" seems to work.

What I've tried: 
a) deleting the node_modules and package-lock.json and running npm istall again
b) running VS code as admin (I'm using the terminal there)
c) installing gatsby-cli as admin
d) using a different starter 

### Reproduction Link

https://github.com/turbodude/gatsby-starter-blog.git

### Steps to Reproduce

run "gatsby develop"

### Expected Result

no errors

### Actual Result

errors

### Environment

```shell
OS: Windows 11 
CPU: Intel Core i7-6400T
Node: v16.16.0
NPM: v8.11.0
```

**Comment by LekoArts** [comment]
You're seeing "There was a problem loading the local develop command" because in step 2 your installation was not successful. If I understand it correctly you didn't fix this  — you'll need to fix this first and then your `node_modules` will contain everything to run Gatsby.

**Comment by turbodude** [comment]
OK, please suggest how to fix that.

**Comment by turbodude** [comment, marked as answer]
I found the solution. 🙏 "Internet Information Services" in OS settings was turned off. Case closed
